test/OrchardCore.Tests/Modules/OrchardCore.Tenants/Services/TenantValidatorTests.cs

```diff
@@ -22,6 +22,7 @@ public class TenantValidatorTests
         [Theory]
         [InlineData("Tenant1", "tenant1", "", "Feature Profile", new[] { "A tenant with the same name already exists." })]
         [InlineData("Tenant5", "tenant3", "", "Feature Profile", new[] { "A tenant with the same host and prefix already exists." })]
+        [InlineData("Tenant5", "tenant3", null, "Feature Profile", new[] { "A tenant with the same host and prefix already exists." })]
```

Comment:
Th third parameter here is `null`, for others, `""`; similarly for other parameters. This needs to be consistent in the `InlineData`s but the validation service should handle both as empty.

Reply:
I just added this to make it clear that the service handles both `null` and `String.Empty`